covid_graphs/covid_graphs/country_graph.py

```diff
@@ -173,7 +180,8 @@ def show_country_plot(data_dir: Path, country_name: str):
     country_report = create_report(data_dir / f"{country_name}.data", short_name=country_name)
 
     for until_date in [
-        datetime.date(2020, 4, 11),
+        country_report.dates[-12],
+        country_report.dates[-7],
```

Comment:
Smart :)

Reply:
:grinning: Yeah, I was trying something. Do you want me to revert this? I do find it useful.